src/coreclr/nativeaot/Common/src/Internal/Runtime/MethodTable.cs

```diff
@@ -506,20 +506,19 @@ internal bool IsGenericTypeDefinition
             get
             {
                 Debug.Assert(IsGeneric);
+
+                uint offset = GetFieldOffset(EETypeField.ETF_GenericDefinition);
```

Comment:
This avoids duplication of the code that computes the offset between the relative and absolute paths. We have used this pattern in a few places already.